### PR TITLE
SEO-172844-WPF-weblink-validatetor-image-

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes in image name and folder image names, please review.
Thanks.
Regards,
Edith.

### CODE REVIEW

File: wpf/DataGrid/Columns.md [hunk 1]
```diff
@@ -928,7 +928,7 @@ You can freeze the columns in view at the left and right side like in excel by s
 {% endhighlight %}
 {% endtabs %}
 
-![Frezzing Columns in WPF DataGrid](Columns_images/wpf-datagrid-freezing-column.png)
```
Comment (MallikaRK) [inline wpf/DataGrid/Columns.md:931]: why you removing column name here. This page descripes, WPF DataGrid Column features, but you are removing column name...revert your changes, Just change casing in image folder.

Check in all places.

File: wpf/Color-Picker/Overview.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 
  WPF [ColorPicker](https://www.syncfusion.com/wpf-ui-controls/colorpicker ) is an user interface to select and adjust color values. This supports various color specifications like RGB (Red Green Blue), HSV (Hue Saturation Value), and Hex codes.
 
- ![Syncfusion WPF ColorPicker-ColorEdit](Getting-Started_images/ColorEdit_Overview.png)
+ ![Syncfusion WPF Color Picker-ColorEdit](getting-started_images/wpf-color-picker-overview-color-edit.png)
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply wpf/Color-Picker/Overview.md:14]: Resolved

File: wpf/DataGrid/Columns.md [hunk 3]
```diff
@@ -773,7 +773,7 @@ You can allow end-users to rearrange the columns by drag and drop the column hea
 {% endhighlight %}
 {% endtabs %}
 
-![Drag and Drop Column in WPF DataGrid](Columns_images/wpf-datagrid-drag-and-drop-columns.png)
+![Drag and Drop Column in WPF DataGrid](columns_images/wpf-datagrid-drag-and-drop.png)
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply wpf/DataGrid/Columns.md:776]: Resolved.

File: wpf/DataGrid/Columns.md (hunk 1 above)
Comment (Edithabich) [reply wpf/DataGrid/Columns.md:931]: Resolved.

File: wpf/DataGrid/Columns.md [hunk 4]
```diff
@@ -547,7 +547,7 @@ public string CustomerID
 
 The OrderID and CustomerID column rearranged based on specified order.
 
-![Changing Columns Order in WPF DataGrid](Columns_images/wpf-datagrid-columns-order.png)
+![Changing Columns Order in WPF DataGrid](columns_images/wpf-datagrid-order.png)
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply wpf/DataGrid/Columns.md]: Resolved

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are  fine.

**Pull request merged**